### PR TITLE
Add '_filter' to filter <tr> elements to fix CSS collision with Bootstrap

### LINKED ISSUE #28: Using "Label" for column name conflicts with default bootstrap CSS
see: https://github.com/level12/agari-adm/issues/147

### PR DESCRIPTION
**Pull request by JSparksman**
Fixes #28

### CODE REVIEW

File: webgrid/static/webgrid.js [hunk 1]
```diff
@@ -105,7 +105,9 @@ function datagrid_add_filter() {
 function datagrid_prep_filters(){
     $('.datagrid .filters tr').each(function(){
         jq_tr = $(this);
-        filter_key = jq_tr.attr('class');
+        // Added _filter to address CSS collision with Bootstrap
+        // Ref: https://github.com/level12/webgrid/issues/28
+        filter_key = jq_tr.attr('class').replace('_filter','');
```
Comment (bladams) [inline webgrid/static/webgrid.js]: NIT: I think a regular expression would be better here so we can avoid potential problems with using the substring `_filter` in our column labels.

**Comment by rsyring** [comment]
From Slack:

@guruofgentoo I agree with @bladams, we are abusing the `class` attribute.  Using a data element would be better, but when I built WebGrid, I was still trying to conform to the xhtml spec.  The better solution overall would be to come up with a better UI for this using React, but I'm not sure that will happen anytime soon.  Are you comfortable moving forward with Jason's quick fix?  Or should we byte the bullet now and have Jason work on removing the abuse of `class`?

**Comment by rsyring** [comment]
@bladams or @guruofgentoo with Matt's change to the regex are we good to merge this PR?  I'd like to get the downstream issue fixed for Agari this sprint, but am waiting on this.

**Comment by bladams** [comment]
I'm fine with merging it as is.

**Pull request merged**